src/main/kotlin/frc/robot/subsystems/climb/ClimbConstants.kt

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems.climb
+
+import com.ctre.phoenix6.configs.MotorOutputConfigs
+import com.ctre.phoenix6.configs.TalonFXConfiguration
+import com.ctre.phoenix6.signals.InvertedValue
+import com.ctre.phoenix6.signals.NeutralModeValue
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.MutableMeasure
+import edu.wpi.first.units.Units
+
+object ClimbConstants {
+    const val GEAR_RATIO = 12.0
+    const val STOPPER_MOTOR_CURRENT_LIMIT = 20
+    const val STOPPER_MOTOR_CURRENT_THRESHOLD = 15 //TODO: calibrate
```

Comment:
Threshold of what? The name is too general.

Reply:
It's the threshold of current for the motor stop at. The BAG motor doesn't have an encoder so it stops based on a current spike when it hits something.

Reply:
Great, but it's not obvious from the constant name.

Reply:
I can't think of a better name for this that wouldn't be an entire sentence.

Reply:
Maybe current limit? It's basically the same as current limits on other motors

Reply:
But It's different from the current limit

Reply:
Ok my bad now I understand. I would add a comment then

Reply:
db0c4fc9c211256ae5829a34f1e6045ed951b576